js/main.js

```diff
@@ -89,6 +88,8 @@ function renderEntry(entry) {
 
   const $penIcon = document.createElement('i');
   $penIcon.className = 'fa-solid fa-pen';
+  $penIcon.setAttribute('data-entry-id', entry.entryId);
+  $penIcon.addEventListener('click', editPencil);
```

Comment:
You shouldn't be adding any eventListeners to the `$penIcon`. We want to use event bubbling on the `ul` element to capture any events that happen on it's children